### ISSUE TITLE
No screen session found.

### ISSUE THREAD

**Issue by Wolfnight29** [devryan/GamePanelX-V3#178]
I have master installed, one remote.

I create a csgo server and the server dont start always offline in console i take:
"No screen session found."

how i can fix?

**Comment by will2therich** [comment]
Hi @Wolfnight29 
Do you have screen installed on your system?
Ta

**Comment by Wolfnight29** [comment]
yes i have gnu screen

**Comment by will2therich** [comment]
Hmm are you only receiving the error when starting a server?
Can you create a template without issues?
And to confirm you are using the remote server for this or a local generated server?

**Comment by Wolfnight29** [comment]
yes i take error when i start server, yes i caan crete template wwithout issues
i create template in remote and i crete server in remote
